app/src/main/java/com/brainwallet/ui/composable/bottomsheet/LanguageSelectorBottomSheet.kt

```diff
@@ -0,0 +1,58 @@
+package com.brainwallet.ui.composable.bottomsheet
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.CheckCircle
+import androidx.compose.material3.Icon
+import androidx.compose.material3.ListItem
+import androidx.compose.material3.ListItemDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import com.brainwallet.data.model.Language
+import com.brainwallet.ui.composable.BrainwalletBottomSheet
+import com.brainwallet.ui.theme.BrainwalletTheme
+
+@Composable
+fun LanguageSelectorBottomSheet(
```

Comment:
Thanks for adding this up for launch!
Hope to add something like this @andhikayuana 
https://medium.com/@nbtk123/create-your-own-horizontal-vertical-slider-picker-android-94b6ee32b3ff
But in vertical...like this:
| Ideal  Picker |
|-------------|
![Screenshot 2025-02-25 at 18 33 41](https://github.com/user-attachments/assets/67bee7c3-65f4-4e5d-9899-304c3ad465a3)|

Reply:
ah okay, noted!